### ISSUE TITLE
No Trace in weblog

### ISSUE THREAD

**Issue by vktrrdk** [nextflow-io/nf-weblog#7]
I tried to get the `trace` from the json sent to the weblog endpoint, but it is not there for me.
I used the latest version of nextflow and also added the corresponding information to the nextflow config.
```
weblog {
  enabled = true
}
trace {
  enabled = true
  fields = 'task_id,name'
}
```

Those are the configurations i tried it with, but the trace was not part of the weblog. Is there anything i have been missing?

**Comment by tuncayseker** [comment]
I previously opened an issue at https://github.com/nextflow-io/nf-weblog/issues/6. However, it appears that the issue has not been resolved. you can try with the earlier versions as I specified in the issue.

**Comment by bentsherman** [comment]
Fixed in 1.1.2
